### ISSUE TITLE
NoSuchMethodError in ZipkinReporter.convertSpan in Play2.8 app

### ISSUE THREAD

**Issue by Cereopsis** [kamon-io/kamon-zipkin#14]
### Setup

java version "1.8.0_221"
Java(TM) SE Runtime Environment (build 1.8.0_221-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.221-b11, mixed mode)

```shell
sbt new playframework/play-scala-seed.g8 --name=play-zipkin-2.8
cd play-zipkin-2.8
```

Append the following to`project/plugins.sbt`:

```scala
addSbtPlugin("io.kamon" % "sbt-kanela-runner-play-2.8" % "2.0.6")
addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent" % "0.1.5")
```

build.sbt:

```scala
lazy val root = (project in file("."))
  .enablePlugins(PlayScala, JavaAgent)
  .settings(
    scalaVersion := "2.13.1",
    javaAgents += "io.kamon" % "kanela-agent" % "1.0.3",
    libraryDependencies	++= Seq(
      guice,
      "io.kamon"   %% "kamon-bundle"  % "2.0.6",
      "io.kamon"   %% "kamon-zipkin"  % "2.0.1"
    )
  )
```

### Optional: Run a basic Zipkin service

```shell
docker run -d -p 9411:9411 openzipkin/zipkin-slim
```

Build & run
```shell
sbt clean playUpdateSecret universal:packageZipTarball
cd target/universal
tar xzf play-zipkin-2.8.tgz
cd play-zipkin-2.8
bin/play-zipkin-2.8
```

Run some load 
```shell
$> for i in {1..100}; do curl localhost:9000/ ;done
```

At some point you will see the following stack trace appear in the terminal 
```
[error] k.m.ModuleRegistry - Reporter [Zipkin Reporter] failed to process a spans tick.
java.lang.NoSuchMethodError: java.lang.Math.floorDiv(JI)J
	at kamon.zipkin.ZipkinReporter.convertSpan(ZipkinReporter.scala:60)
	at kamon.zipkin.ZipkinReporter.$anonfun$reportSpans$1(ZipkinReporter.scala:57)
	at scala.collection.immutable.List.map(List.scala:219)
	at scala.collection.immutable.List.map(List.scala:79)
	at kamon.zipkin.ZipkinReporter.reportSpans(ZipkinReporter.scala:57)
	at kamon.module.ModuleRegistry$$anon$2.$anonfun$run$4(ModuleRegistry.scala:214)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:661)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:430)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
```
 
Probing a little in sbt console, there is no such problem with doing pretty much exactly what `ZipkinReporter.convertSpan` appears to be doing.

**Comment by jatcwang** [comment]
We're not using Play and we're getting the same issue. Running the app with JDK 11 (`openjdk:11` as the base docker image) seems to fix the issue..

**Comment by dpsoft** [comment]
Fixed with #15
